Cargo.toml

```
@@ -75,10 +71,6 @@ features = ["macros", "libadwaita", "gnome_43"]
 version = "1.0.126"
 features = ["derive"]
 
```

This may fix the issue

```suggestion

[dependencies.gettext-rs]
version = "0.7"
features = ["gettext-system"]
```

---

thanks, I added it back and it builds now

I think this might be a cargo-machete bug, I will file a bug upstream